public/sass/elements/_reset.scss

```diff
@@ -117,3 +117,7 @@ td {
   line-height: inherit;
   font-weight: normal;
 }
+
+abbr[title], acronym[title] {
```

Comment:
Could this be achieved without the title attribute selectors? This would enable people to style an abbr or acronym by putting a selector directly on it.